apps/docs/pages/docs/api-reference/components/drop-zone.mdx

```diff
@@ -36,6 +36,7 @@ const config = {
 | [`minEmptyHeight`](#minemptyheight) | `minEmptyHeight: 256`        | Number        | -        |
 | [`ref`](#ref)                       | `ref: ref`                   | Ref           | -        |
 | [`style`](#style)                   | `style: {display: "flex"}`   | CSSProperties | -        |
+| [`as`](#as)                         | `as: "section"`              | ElementType   | -        |
```

Comment:
The primary sort is status, and secondary is alphabetical. Can you move this under `allow`?